.gitignore

```diff
@@ -281,3 +281,6 @@ __pycache__/
 
 #generated localization files
 Localization/
+
+/DFC.ServiceTaxonomy.Theme/wwwroot/Styles/trumbowyg_scoped_govuk_frontend.css
```

Comment:
Would you mind adding these files back into git, as it's useful to see their history of changes.